internal/acceptance/sql_table_test.go

```diff
@@ -0,0 +1,191 @@
+package acceptance
+
+import (
+	"testing"
+)
+
+func TestUcAccResourceSqlTable_Managed(t *testing.T) {
+	unityWorkspaceLevel(t, step{
+		Template: `
+		resource "databricks_schema" "this" {
+			name 		 = "foom"
+			catalog_name = "main"
+		}
+
+		resource "databricks_sql_table" "this" {
+			name               = "bar"
+			catalog_name       = "main"
+			schema_name        = databricks_schema.this.name
+			table_type         = "MANAGED"
+			properties         = {
+				this      = "that"
+				something = "else"
+			}
+
+			column {
+				name      = "id"
+				type      = "int"
+			}
+			column {
+				name      = "name"
+				type      = "varchar(64)"
+			}
+			comment = "this table is managed by terraform"
+		}`,
+	}, step{
+		Template: `
+		resource "databricks_schema" "this" {
+			name 		 = "foom"
+			catalog_name = "main"
+		}
+
+		resource "databricks_sql_table" "this" {
+			name               = "bar"
+			catalog_name       = "main"
+			schema_name        = databricks_schema.this.name
+			table_type         = "MANAGED"
+			properties         = {
+				that      = "this"
+				something = "else2"
+			}
+			
+			column {
+				name      = "id"
+				type      = "int"
+			}
+			column {
+				name      = "name"
+				type      = "string"
+			}
+			comment = "this table is managed by terraform..."
+		}`,
+	})
+}
+
+func TestUcAccResourceSqlTable_External(t *testing.T) {
+	unityWorkspaceLevel(t, step{
```

Comment:
make it AWS-specific
```suggestion
        GetEnvOrSkipTest(t, "TEST_EC2_INSTANCE_PROFILE")
	unityWorkspaceLevel(t, step{
```